src/main/java/com/dubture/jenkins/digitalocean/SlaveTemplate.java

```diff
@@ -111,16 +112,18 @@
      * @param sizeId the image size e.g. "512mb" or "1gb"
      * @param regionId the region e.g. "nyc1"
      * @param idleTerminationInMinutes how long to wait before destroying a slave
+     * @param numExecutors the number of executors that this slave supports
      * @param labelString the label for this slave
      */
     @DataBoundConstructor
-    public SlaveTemplate(String imageId, String sizeId, String regionId, String idleTerminationInMinutes, String labelString) {
+    public SlaveTemplate(String imageId, String sizeId, String regionId, String idleTerminationInMinutes, String numExecutors, String labelString) {
         LOGGER.log(Level.INFO, "Creating SlaveTemplate with imageId = {0}, sizeId = {1}, regionId = {2}", new Object[] { imageId, sizeId, regionId});
         this.imageId = imageId;
         this.sizeId = sizeId;
         this.regionId = regionId;
 
         this.idleTerminationInMinutes = Integer.parseInt(idleTerminationInMinutes);
+        this.numExecutors = Integer.parseInt(numExecutors);
```

Comment:
For all Templates created bevore applying this patch this value is null which results in a nullpointer exception. We might want to check for null and use 1 instead.